ext/calendar/tests/easter_date_checks_upper_bound_64bit.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Test easter_date() on 64bit systems checks the upper year limit
+--SKIPIF--
+<?php if (PHP_INT_SIZE != 8) die("skip 64-bit only"); ?>
+--INI--
+date.timezone=UTC
+--ENV--
+TZ=UTC
+--EXTENSIONS--
+calendar
+--FILE--
+<?php
+putenv('TZ=UTC');
+try {
+    easter_date(293274701009);
+} catch (ValueError $ex) {
+    echo "{$ex->getMessage()}\n";
+}
+?>
+--EXPECT--
+easter_date(): Argument #1 ($year) must be a year after 1970 (inclusive)
```

Comment:
In this case, the error message could be confusing. The method call is `easter_date(293274701009);` ... so it's "correct" because the error message only mentions that the argument must be `> 1970` and that's true in this case.

Maybe we should mention the upper limit too?
(Or maybe we should not because, honestly, who needs to know the Easter date of the year 2 billion?)

Reply:
I am also really split on this. I did leave it out because it just looked ridiculous, but if there is an argument to add the year, I will add it.